CHANGES.rst

```diff
@@ -106,6 +106,10 @@ tweakreg
   additional user-provided rotations and scale corrections to an imaging
   WCS of a calibrated image. [#7430]
 
+1.9.5 (2023-03-02)
+==================
+
+- add ``opencv-python`` to ``requirements-sdp.txt``
```

Comment:
Whoops, need to reference a PR number here to make the changelog bot happy.

Reply:
this change doesn't have a PR number, but I can add the label to this PR

Reply:
Or just add the "no-changelog-entry-required" label.

Reply:
Ah, I see you beat me to it!